### PR TITLE
Initial replacement of old style connect syntax (with PMFs)

### PR DESCRIPTION
**Pull request by iamthesenate1**
This PR does not include changes that need other changes to the code.

### CODE REVIEW

**Comment by tsujan** [comment]
Reviewing this will take time. In future, please split big patches into small PRs; they minimize the possibility of mistakes. But please don't change this PR — I started to review it, little by little.

**Comment by iamthesenate1** [comment]
Ok. Thank's for the suggestion.

File: src/qps.cpp [hunk 1]
```diff
@@ -996,9 +996,9 @@ void Qps::make_command_menu()
     if (flag_devel)
     {
         m_command->addAction( tr( "WatchDog" ), watchdogDialog,
-                             SLOT(show()));
+                             &Qps::show);
```
Comment (tsujan) [inline src/qps.cpp:999]: In all places up to this point, you followed what I'd call good practice and used signals/slots of original widgets when they weren't overridden. Please use `&QWidget::show` here for the sake of consistency, in a separate commit and without squashing (for me to continue the review).

Comment (iamthesenate1) [reply src/qps.cpp]: Done!

**Review by tsujan — APPROVED** [review]
Everything seems fine. Thanks!

**Pull request merged**